Common/doc/docmain.hpp

```diff
@@ -15,7 +15,7 @@
  *  - Prof. Edwin van der Weide's group at the University of Twente.
  *  - Prof. Vincent Terrapon's group at the University of Liege.
  *
- * Copyright 2012-2021, Francisco D. Palacios, Thomas D. Economon,
+ * Copyright 2012-2022, Francisco D. Palacios, Thomas D. Economon,
```

Comment:
These are all still using the old header format, that's why I didn't catch them. Can you upgrade these files?
They should be "auto" updated after that.

Reply:
yup.  Should I remove the portions that mention the main research groups?
`| The main research teams contributing to the current release are:      |
| - Prof. Juan J. Alonso's group at Stanford University.                |
| - Prof. Piero Colonna's group at Delft University of Technology.      |
| - Prof. Nicolas R. Gauger's group at Kaiserslautern U. of Technology. |
| - Prof. Alberto Guardone's group at Polytechnic University of Milan.  |
| - Prof. Rafael Palacios' group at Imperial College London.            |
| - Prof. Vincent Terrapon's group at the University of Liege.          |
| - Prof. Edwin van der Weide's group at the University of Twente.      |
| - Lab. of New Concepts in Aeronautics at Tech. Inst. of Aeronautics.`

Reply:
This should be the common "middle part", between the \authors and the license stuff.
```
 * \version 7.3.0 "Blackbird"
 *
 * SU2 Project Website: https://su2code.github.io
 *
 * The SU2 Project is maintained by the SU2 Foundation
 * (http://su2foundation.org)
 *
 * Copyright 2012-2022, SU2 Contributors (cf. AUTHORS.md)
 *
 * SU2 is free software; ...
```

Reply:
Okay, Ill update the files accordingly.